modules/pages-and-forms/pages/widgets.adoc

```diff
@@ -42,6 +42,8 @@ Limitations:
 * The URL should have the file type as extension (e.g. \https://myhost/doc/sample.pdf)
 ====
 
+See [#Troubleshooting UI Designer widgets] to solve common issues with this widget.
```

Comment:
```suggestion
See <<troubleshooting-ui-designer-widgets>> to solve common issues with this widget.
```

Example could be found here, https://bonitasoft.github.io/bonita-documentation-theme/links.html#_anchors_to_navigate_inside_the_same_document